src/Common/Auth/MfaUtils.php

```diff
@@ -25,8 +25,7 @@ class MfaUtils
     private $regs;
     private $registrations;
     private $var1U2F;
-    private $var1TOTP;
-    private $uid; // User Id who try connect
+    private $var1TOTP; // User Id who try connect
```

Comment:
weird, comment went to wrong place

Reply:
Yeah, that happened a bunch. I've been trying to clean it up.